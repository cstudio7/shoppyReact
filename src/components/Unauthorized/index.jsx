import React, { Component, Fragment } from 'react';
import { Link } from 'react-router-dom';
import '../NotFound/NotFound.scss';

export default class Unauthorized extends Component {
  render() {
    return (
      <Fragment>
        <div id="notfound">
          <div className="notfound">
            <div className="notfound-404">
              <h1>Oops!</h1>
              <h2>You Dont Have Access To View This Page</h2>
            </div>
            <Link to="/">
              <p>Go TO Homepage</p>
            </Link>
          </div>
        </div>
      </Fragment>
    );
  }
}
