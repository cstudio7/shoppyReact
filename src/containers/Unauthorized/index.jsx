import React, { Component } from 'react';
import Unauthorized from '../../components/Unauthorized';

export default class UnauthorizedPage extends Component {
  render() {
    return <Unauthorized />;
  }
}
