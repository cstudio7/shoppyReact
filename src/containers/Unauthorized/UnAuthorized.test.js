import React from 'react';
import { shallow } from 'enzyme';
import Unauthorized from './index';

describe('Test the Unauthorized Container', () => {
  it('should match snapshot', () => {
    const wrapper = shallow(<Unauthorized />);
    expect(wrapper).toMatchSnapshot();
  });
});
