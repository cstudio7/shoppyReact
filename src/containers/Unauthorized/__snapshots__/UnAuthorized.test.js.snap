// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Test the Unauthorized Container should match snapshot 1`] = `<Unauthorized />`;
